scripts/get-e2e-db-schema.ts

```diff
@@ -26,24 +28,36 @@ const connectionStringMapping: Record<ComboCase, string> = {
 const args = arg({
   '--os': String,
   '--node-version': String,
+  '--prisma-client-version': String,
   '--github-env': String,
 })
 
-const comboCase = parseComboCase(args['--node-version'] ?? '', args['--os'] ?? '')
+const schemaName = parseComboCase(
+  args['--node-version'] ?? '',
+  args['--os'] ?? '',
+  args['--prisma-client-version'] ?? ''
+)
 
 if (args['--github-env']) {
-  fs.append(args['--github-env'], `E2E_DB_SCHEMA=${connectionStringMapping[comboCase]}`)
+  fs.append(args['--github-env'], `E2E_DB_SCHEMA=${schemaName}`)
 } else {
-  process.stdout.write(connectionStringMapping[comboCase])
+  process.stdout.write(schemaName)
 }
 
 //
 // Helpers
 //
 
-function parseComboCase(nodeVersionInput: string, osInput: string): ComboCase {
+function parseComboCase(nodeVersionInput: string, osInput: string, prismaClientInput: string): string {
   const nodeVersion = nodeVersionParser.parse(nodeVersionInput)
   const os = osParser.parse(osInput)
   // eslint-disable-next-line
-  return [nodeVersion, os].join(' + ') as any
+  const comboCase = [nodeVersion, os].join(' + ') as ComboCase
```

Comment:
This could be done in a type safe way using zod. If you make `nodeVersion` and `os` be unions of string literals instead of strings you'll see the cast is no longer needed.

If you want to go all-in on type-safety you can also maintain a zod list of prisma client versions––there are in the end not so many versions to maintain nor changing often.

Reply:
I got a better result by using it as a string.

```js 
const comboCase: ComboCase = `${nodeVersion} + ${os}`
```
About the prisma version, I fixed it to work only with the `3.x` versions